etc/deploy/Dockerfile

```diff
@@ -1,28 +1,28 @@
 FROM jorge07/alpine-php:7.2-dev-sf as builder
 
-WORKDIR /build
+WORKDIR /app
 
 ENV APP_ENV prod
+ENV APP_SECRET default-secret
+ENV RABBITMQ_URL amqp://guest:guest@rmq:5672
 
-COPY composer.json /build
-COPY composer.lock /build
-COPY symfony.lock /build
+COPY composer.json /app
+COPY composer.lock /app
+COPY symfony.lock /app
 
 RUN composer install --no-ansi --no-scripts --no-dev --no-interaction --no-progress --optimize-autoloader
 
-COPY bin /build/bin
-COPY config /build/config
-COPY src /build/src
-COPY public /build/public
+COPY bin /app/bin
```

Comment:
Just for my understanding, what's the benefit of doing this in 2 steps ... 

1. load vendors
2. run scripts

Reply:
Caching.

Docker invalidates the cache of the following layers if the current ones change. 

Doing this I'm sure that I'll only need to run composer install if dependencies change.

Reply:
Understood. Clever!